p/scripts/main.js

```diff
@@ -2076,9 +2076,6 @@ function init_normal() {
 		if (sidebar) {	// Save sidebar scroll position
 			sessionStorage.setItem('FreshRSS_sidebar_scrollTop', sidebar.scrollTop);
 		}
-		if (mark_read_queue && mark_read_queue.length > 0) {
-			return false;
```

Comment:
The goal of this line is to prevent leaving the page if there are pending requests (such as several articles to be marked as read) which have not completed.
Leaving too early would break some of the user's actions